src/GOODFFunctions.h

```diff
@@ -62,7 +68,7 @@ namespace GOODF_functions {
 	inline wxString removeBaseOdfPath(wxString path) {
 		wxString stringToReturn = path;
 		wxFileName fName = wxFileName(path);
-		if (fName.FileExists()) {
+		if (fName.FileExists() && stringToReturn.StartsWith(wxFILE_SEP_PATH)) {
```

Comment:
I'm not certain here, but I don't think that the string starting with a file separator should be part of the condition as the removeBaseOdfPath is used for a few other things too, like when the write location of the .organ file would be another from where it was read. Cleaning out the file separator, if existing, is a later step but the earlier could potentially still be needed. I would need further testing to be sure though.

Reply:
Without this test, when loading a rank with attacks and release GoOdf writes a .organ file that contains Pipe999Release999=`<path>` where path begins with "`../somewhere`" and  "`../somewhere`" doesn't exist.  Each time I created or modified a .organ file I needed to hand edit the .organ file to fix the `<path>`.   I'll see if I can put together a **reproducible** test

( In my environment, ${HOME}/GrandOrgue is a symbolic link to /tank/GOfiles/GrandOrgue (tank/GOfiles being a ZFS filesystem) this might be causing confusion with WX. )

Reply:
Strange. I've used plenty of ODFs coming from OdfEdit that contain ../some/path and they all work fine in GoOdf when modified or just loaded.

Reply:
The good news is that, despite my best efforts, I've been **unable** to duplicate the error I was seeing in saving files!  It's probably safe to revert that change.  I'll revisit if I'm ever able to provide a simple test confirming the issue.

One thing I notice, though,  is that the path separator is different in the Attack and Release sections on loaded .organ files: Attack(s) have `/`, Release(s) `\`   (Related to the issue I was having?  Perhaps.)
![Slash](https://github.com/user-attachments/assets/1686d1c1-8115-44d1-a27a-d1d42b4693bf)

When you initially load the samples into the organ, both Attack(s) and Release(s) have `/` as path separator.
![Load](https://github.com/user-attachments/assets/5dd1586e-739a-4d0d-b8ce-813d9af2077a)